### PR TITLE
Replaced Cathy with Denzel

### PR DESCRIPTION
**Pull request by Lenzed26**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -7,7 +7,7 @@ class Program
         static void Main(string[] args)
         {
             Console.WriteLine("Hello World!");
-			Console.WriteLine("Hello from Cathy French");
+			Console.WriteLine("Hello from Denzel");
```
Comment (keags14) [inline Program.cs:10]: This looks great the unit tests passed.

**Review by keags14 — APPROVED** [review]
I have tested the new code and it works as expected.

File: Program.cs (hunk 1 above)
Comment (Lenzed26) [reply Program.cs:11]: Thank you

**Review by tak-li98 — APPROVED** [review]
Perfect!

**Review by Jayciee — APPROVED** [review]
Your hello has been approved

**Review by atchu98 — APPROVED** [review]

**Pull request merged**